server/webhooks/__init__.py

```diff
@@ -0,0 +1,60 @@
+from urllib.parse import urlparse
+from server.atl_requests import Context
+from flask import url_for
+
+
+class WebhookContext(Context):
```

Comment:
Not sure this needs to be its own class. Presumably you can just create a `Context` and move the `parse_callback` call to a standalone helper function in the module.